src/subprocess.h

```diff
@@ -52,7 +52,7 @@ struct Subprocess {
 
  private:
   Subprocess(bool use_console);
-  bool Start(struct SubprocessSet* set, const std::string& command);
+  bool Start(struct SubprocessSet* set, const std::string& command, int priority);
```

Comment:
Please document the meaning of `priority` here (including range of valid values and their interpretation, in particular for Posix _and_ Windows).

Reply:
Is the documentation I added sufficient?

Reply:
Yes, thank you.